regression/goto-harness/nondet_strings/main.c

```diff
@@ -0,0 +1,6 @@
+#include <assert.h>
```

Comment:
:pick: can you split the commit to have one with only the tests

Reply:
no

Reply:
A test always makes sense, even if it is failing. Having it in a separate commit makes it easier to cherry pick and check it was failing before, or on another branch.